crates/bevy_sprite/src/mesh2d/mesh.rs

```diff
@@ -39,7 +39,7 @@ use crate::Material2dBindGroupId;
 /// Component for rendering with meshes in the 2d pipeline, usually with a [2d material](crate::Material2d) such as [`ColorMaterial`](crate::ColorMaterial).
 ///
 /// It wraps a [`Handle<Mesh>`] to differentiate from the 3d pipelines which use the handles directly as components
-#[derive(Default, Clone, Component, Debug, Reflect, PartialEq, Eq)]
+#[derive(Default, Clone, Component, Debug, Reflect, PartialEq, Eq, Deref)]
```

Comment:
Any reason to not derive `DerefMut` as well?

Reply:
I can added DerefMut,but others maybe have any reason?

Reply:
I think we can derive DerefMut too, maybe its unexpected to have normal deref and not have DerefMut